### PR TITLE
Flashers now use flashes instead of snowflake stun

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

Flashers now inherit the behavior of flash assemblies by directly calling their `flash_carbon` and `flash_borg` (new) procs. This has no negative effect on the flash's lifespan.

Mounted flashers will still apply twice the "strength" of a normal flash and thus apply twice as many confusion stacks. (hellish)

# Changelog

:cl:  
tweak: Mounted and portable flashers no longer have special stun behavior from flash assemblies
/:cl:

### CODE REVIEW

File: code/modules/assembly/flash.dm [hunk 1]
```diff
@@ -171,32 +171,36 @@
 		if(M.flash_act())
 			M.set_confusion_if_lower(power * CONFUSION_STACK_MAX_MULTIPLIER SECONDS)
 
+/obj/item/assembly/flash/proc/flash_borg(mob/living/silicon/robot/R, mob/user)
+	log_combat(user, R, "flashed", src)
+	if(!R.sensor_protection)
+		update_icon(ALL, flash = TRUE)
+		R.Paralyze(rand(80,120))
```
Comment (tattax) [inline code/modules/assembly/flash.dm]: time defines

File: code/modules/assembly/flash.dm [hunk 2]
```diff
@@ -171,32 +171,36 @@
 		if(M.flash_act())
 			M.set_confusion_if_lower(power * CONFUSION_STACK_MAX_MULTIPLIER SECONDS)
 
+/obj/item/assembly/flash/proc/flash_borg(mob/living/silicon/robot/R, mob/user)
```
Comment (tattax) [inline code/modules/assembly/flash.dm]: var names

File: code/game/machinery/flasher.dm [hunk 3]
```diff
@@ -124,7 +124,10 @@
 			continue
 
 		if(L.flash_act(affect_silicon = 1))
```
Comment (JohnFulpWillard) [inline code/game/machinery/flasher.dm:126]: what does this do that we need a second flash_borg too?

Comment (ynot01) [reply code/game/machinery/flasher.dm:126]: flash_act only interacts with the eyes- it blurs/blinds/applies the flash overlay and also checks for flash protection, returning whether it was successful or not

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**